content/02-understand-prisma/06-api-comparisons/01-prisma-and-mongoose.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: 'Comparing Prisma and Mongoose'
+metaTitle: 'Comparing Prisma and Mongoose'
+metaDescription: 'Compare the Prisma and Mongoose APIs.'
+---
+
+## Overview
+
+This page compares the Prisma and [Mongoose](https://mongoosejs.com/docs/guide.html) APIs.
+
+
+## Fetching single objects
+
+**Prisma**
+
+```ts
+const user = await prisma.user.findOne({
+  where: {
+    id: 1,
+  },
+});
+```
+**Mongoose**
+
+```ts
+  const result = await User.findById("5eb9354624286a04e42401d8");
+```
+
+## Fetching selected scalars of single objects
+
+**Prisma**
+
+```ts
+const user = await prisma.user.findOne({
+    where: {
+      id: 1,
+    },
+    select: {
+      name: true,
+    },
+  });
+```
+
+**Mongoose**
+
+```ts
+  const user = await User.findById("5eb9354624286a04e42401d8").select([
+    "name",
+    "email",
+  ]);
+```
+
+## Fetching relations
+
+**Prisma**
+<Code hideCopy={true} languages={["Using include", "Fluent API"]}>
```

Comment:
isn't the MDX component called `CodeBlock`? does this render properly?

Reply:
Fixed!